### PR TITLE
Update duckduckgo.po ja_JP

### PR DESCRIPTION
**Pull request by AKKED**
Add and update translation.

### CODE REVIEW

File: locales/ja_JP/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -1076,7 +1076,7 @@ msgstr "もしかして：%s"
 #. Non lle poño "enderezos" pois non se refire aos enderezos se non as instruccións para chegar a un lugar
 msgctxt "maps_places"
 msgid "Directions"
-msgstr "位置情報"
+msgstr "地図を開く"
```
Comment (tagawa) [inline locales/ja_JP/LC_MESSAGES/duckduckgo.po]: Hi there. Although this button does open the map, I think the translation for "Directions" should perhaps be something like ナビゲーション or 行き方 or ルート (or something better). What do you think?

Comment (AKKED) [reply locales/ja_JP/LC_MESSAGES/duckduckgo.po]: That's true.
I tried to think again.
I think "ルート検索" is good.
How about this?

Comment (tagawa) [reply locales/ja_JP/LC_MESSAGES/duckduckgo.po]: Oh yes, that's a good idea. ありがとう！

**Comment by tagawa** [comment]
Perfect!
@AKKED いつもありがとうございます。 😄

**Pull request merged**